sematic/ui/packages/common/src/component/GitInfo.tsx

```diff
@@ -36,12 +42,12 @@ const GitInfoBox = () => {
             <Link>pf49df3</Link>
             <CopyButton text={"pf49df3"} />
```

Comment:
When we have real data, the copy button will be for the whole commit sha, not just the trimmed one.

Reply:
Yes, fixed in the static component. Will keep it in mind when actually implementing with real data.